api/views.py

```diff
@@ -28,7 +28,7 @@ def post(self, request, format=None):
 
 class PatientRetrieveUpdateAPIView(APIView):
     serializer_class = PatientSerializer
-    permission_classes = [permissions.IsAuthenticated]
+    # permission_classes = [permissions.IsAuthenticated]
```

Comment:
**PatientRetrieveUpdateAPIView** should be protected. uncomment **permission_classes** attribute.